config/config.exs

```diff
@@ -20,7 +20,8 @@ config :livebook,
   authentication_mode: :password,
   token_authentication: false,
```

Comment:
I think `token_authentication` is no longer relevant. Also you probably don't need the endpoint configuration above, since it should be the default now :)

Reply:
Thanks - I removed `token_authentication`.

I looked briefly into removing the endpoint configuration, but got the following exception:

```
07:19:11.843 [error] #PID<0.10216.0> running LivebookWeb.Endpoint (connection #PID<0.10209.0>, stream id 6) terminated
Server: localhost:8080 (http)
Request: GET /
** (exit) an exception was raised:
    ** (ArgumentError) the signing salt for LivebookWeb.Endpoint is missing or too short.

Add the following LiveView configuration to your config/runtime.exs
or config/config.exs:

    config :my_app, MyAppWeb.Endpoint,
        ...,
        live_view: [signing_salt: "FqsebvoMJuiwQ8Vm"]


        (phoenix_live_view 0.16.0) lib/phoenix_live_view/utils.ex:261: Phoenix.LiveView.Utils.salt!/1
        (phoenix_live_view 0.16.0) lib/phoenix_live_view/static.ex:343: Phoenix.LiveView.Static.sign_token/2
        (phoenix_live_view 0.16.0) lib/phoenix_live_view/static.ex:113: Phoenix.LiveView.Static.render/3
        (phoenix_live_view 0.16.0) lib/phoenix_live_view/controller.ex:35: Phoenix.LiveView.Controller.live_render/3
        (phoenix 1.5.12) lib/phoenix/router.ex:352: Phoenix.Router.__call__/2
        (livebook 0.3.0-dev) lib/livebook_web/endpoint.ex:1: LivebookWeb.Endpoint.plug_builder_call/2
        (livebook 0.3.0-dev) lib/livebook_web/endpoint.ex:1: LivebookWeb.Endpoint.call/2
        (phoenix 1.5.12) lib/phoenix/endpoint/cowboy2_handler.ex:65: Phoenix.Endpoint.Cowboy2Handler.init/4
```

Reply:
Hmm interesting, the salt should be in the `def application` `:env` config:

```elixir
# MIX_ENV=prod iex -S mix

iex(livebook_tzzrfbcx@mockingjay)1> Application.get_all_env(:livebook)
[
  {:file_systems,
   [%Livebook.FileSystem.Local{default_path: "/home/jonatanklosko/_/livebook/"}]},
  {:plugs, []},
  {:cookie, :"YDtXDpL36kJOMMHsf5k2_IYi2L2ih5s6kiaqOTiGwNNYuAHmqPOWZDZ-"},
  {:default_runtime, {Livebook.Runtime.ElixirStandalone, []}},
  {LivebookWeb.Endpoint,
   [
     url: [host: "localhost"],
     pubsub_server: Livebook.PubSub,
     live_view: [signing_salt: "livebook"],
     server: true,
     secret_key_base: "V5+Ep4xa2vcYhJMACEDOavGAEF+B4T8JNHdwS1fdv+PSFdsEx02bS0CjDGx+8fN3",
     http: [ip: {127, 0, 0, 1}, port: 3333]
   ]},
  {:authentication_mode, :token},
  {:token, "nj4xaf4nvmrcxw3mcs3izjy6p4dpyqgw"}
]
```

@josevalim any ideas?